llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1398,6 +1780,79 @@ DebugAbbrevOffsetsRefAdaptor::encodeOffsets(ArrayRef<size_t> Offsets) {
   return EncodedOffsets;
 }
 
+static void
+getFixupsForCuDie(InMemoryCASDWARFObject::SplitUpCUData &SplitData,
```

Comment:
Can you remind me where we left off in the discussion here: do we still need CU fixups if we use the partitioned storage scheme?

Reply:
ah.. these aren't fixups, this is the partitioning algorithm?

Reply:
yes, these are not fixups like the addends, I call them fixups because they go into a `DebugInfoFixupsRef` where the fixups are anything that are deduped and need to be fixed up when we re create the compile unit contents when we materialize, if the name is too confusing, what do you suggest we change it to?